### PR TITLE
Make prefixes only count non-color characters & General cleanup

### PR DESCRIPTION
**Pull request by TechnicallyCoded**
Prefixes count color codes in the max length which isn't actually displayed to the end-user. Therefore the character limit shouldn't apply to color codes.

### CODE REVIEW

File: src/main/java/me/loving11ish/clans/commands/clanSubCommands/ClanPrefixSubCommand.java [hunk 1]
```diff
@@ -24,46 +25,63 @@ public class ClanPrefixSubCommand {
...
             Player player = (Player) sender;
-            clans.forEach((clans) ->
-                    clansPrefixList.add(clans.getValue().getClanPrefix()));
+            clans.forEach((clans) -> clansPrefixList.add(clans.getValue().getClanPrefix()));
+
             if (args.length == 2) {
-                if (bannedTags.contains(args[1])){
-                    player.sendMessage(ColorUtils.translateColorCodes(messagesConfig.getString("clan-prefix-is-banned").replace("%CLANPREFIX%", args[1])));
+                String prefixRaw = args[1];
+                String prefixColorStripped = ChatColor.stripColor(ChatColor.translateAlternateColorCodes('&', prefixRaw));
```
Comment (Loving11ish) [inline src/main/java/me/loving11ish/clans/commands/clanSubCommands/ClanPrefixSubCommand.java:32]: Does this also check for # RGB & HEX code checking/stripping?

**Review by Loving11ish — CHANGES_REQUESTED** [review]
Can you also change the sender.sendMessage() to player.sendMessage() please?

**Comment by TechnicallyCoded** [comment]
sender.sendMessage() was the original. I didn't change anything that wasn't related to the goal.

**Comment by Loving11ish** [comment]
> sender.sendMessage() was the original. I didn't change anything that wasn't related to the goal.

Ok fair enough.  How about the other comment?

**Comment by Loving11ish** [comment]
Now implemented into api dev branch.  Many thanks!

**Pull request closed without merging**